.releaserc.yml

```diff
@@ -0,0 +1,22 @@
+branches:
+  - main
+repositoryUrl: 'git@github.com:liatrio/envelope-game.git'
+preset: conventionalcommits
+plugins:
+  - '@semantic-release/commit-analyzer'
+  - '@semantic-release/release-notes-generator'
+  - path: '@semantic-release/github'
```

Comment:
the ordering of these plugins matters, fyi.  this plugin is going to try to upload release assets to github, but these assets haven't been created yet

Reply:
Added that